src/app/creator-profile-page/creator-profile-details/creator-profile-details.component.ts

```diff
@@ -199,4 +202,22 @@ export class CreatorProfileDetailsComponent {
       this.globalVars.loggedInUser?.UsersYouHODL?.length === 0
     );
   }
+
+  // Set Title function for dynamically setting the title
+  public setTitle(newTitle: string) {
+    this.titleService.setTitle(newTitle);
+  }
+
+  // Set Meta function for dynamically setting the meta tags
+  public setMeta(newContent: string) {
```

Comment:
Sorry, but could you remove the dynamic meta tags as well? They will need to be done differently when we move to server-side rendering.

Reply:
Yes I will just update this other PR: https://github.com/bitclout/frontend/pull/97 which has the titles, metatags were removed